.github/workflows/github.yml

```diff
@@ -72,22 +73,33 @@ jobs:
             conan_prebuilts: dependencies-ios
             conan_options: --options with_apple_system_libs=True
           - platform: msvc-x64
+            arch: x64
             os: windows-latest
```

Comment:
Potential problem in future: right now `windows-latest` points to Win-2022 image. But Win-2025 image comes without innosetup preinstalled.

Meaning, CI might break in future if Github were to switch `latest` image to 2025 or when 2022 image will be deprecated.

Reply:
I have removed download and install Inno Setup step as @kambala-decapitator suggested. Can be added in future. I quess in 2025 runner will be Inno Setup preinstalled when right time commes. Otherwise, we can re-add that step now to be sure.

Reply:
Maybe add just for sure add Inno Setup back?

    - name: Ensure Inno Setup is installed
      if: ${{ startsWith(matrix.platform, 'msvc') }}
      shell: bash
      run: |
        if [ ! -f "/c/Program Files (x86)/Inno Setup 6/ISCC.exe" ] && [ ! -f "/c/ProgramData/Chocolatey/bin/ISCC.exe" ]; then
          choco install innosetup --no-progress -y
        fi